src/PSR7/Validators/SerializedParameter.php

```diff
@@ -181,6 +183,14 @@ protected function convertToSerializationStyle($value, ?CebeSchema $schema)
         if ($schema && $this->style === self::STYLE_DEEP_OBJECT) {
             foreach ($value as $key => &$val) {
                 $childSchema = $this->getChildSchema($schema, (string) $key);
+
+                if (isset($childSchema->oneOf)) {
```

Comment:
Looks like in this case we should immediately stop validation if we have filled oneOf and none of the schemas matched in `findSuitableOneOf`.